tests/test_cloudformation/test_cloudformation_custom_resources.py

```diff
@@ -0,0 +1,121 @@
+import boto3
+import json
+import sure  # noqa # pylint: disable=unused-import
+import time
+
+from moto import mock_lambda, mock_cloudformation, mock_logs, mock_s3, settings
+from pprint import pprint
+from unittest import SkipTest
+from uuid import uuid4
+from tests.test_awslambda.utilities import wait_for_log_msg
+from .fixtures.custom_lambda import get_template
+
+
+def get_lambda_code():
+    pfunc = """
+def lambda_handler(event, context):
+    print(event)
+    response = dict()
+    response["Status"] = "SUCCESS"
+    response["StackId"] = event["StackId"]
+    response["RequestId"] = event["RequestId"]
+    response["LogicalResourceId"] = event["LogicalResourceId"]
+    response["PhysicalResourceId"] = "{resource_id}"
+    response_data = dict()
+    response_data["info_value"] = "special value"
+    if event["RequestType"] == "Create":
+        response["Data"] = response_data
+    print(response)
+    print("finished")
+    import cfnresponse
+    cfnresponse.send(event, context, cfnresponse.SUCCESS, response_data)
+    return response
+""".format(
+        resource_id=f"CustomResource{str(uuid4())[0:6]}"
+    )
+    return pfunc
+
+
+@mock_cloudformation
+@mock_lambda
+@mock_logs
+@mock_s3
+def test_create_custom_lambda_resource():
```

Comment:
This test runs against AWS - I've just reduced the wait times, as AWS sometimes takes 1 minute to even get started with stack creation